### PR TITLE
docs(app): Add Atom IDE setup

### LINKED ISSUE #10: Add Atom IDE setup
Create Atom IDE setup guide

### PR DESCRIPTION
**Pull request by reeddunkle**
- added Atom IDE setup

Closes #10

### CODE REVIEW

File: ide/atom/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+[Packages](#packages) | [Shortcuts](#shortcuts) | [CLI](#cli) | [Feedback](#feedback)
+
+# Atom
+
+Download: <https://atom.io/>
+
+## Packages
```
Comment (chawkins-nj) [inline ide/atom/README.md:7]: Are we going to update this every time this list changes/updates? Maybe we should leave off the version numbers and/or only list the packages that we all use that aren't already installed by default?

File: ide/atom/README.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- fonts@2.0.0 (\*)
+- highlight-selected@0.13.1
+- intentions@1.1.2
+- linter@2.1.2
+- linter-eslint@8.1.4
+- linter-stylelint@3.10.0
+- linter-ui-default@1.2.2
+- minimap@4.27.1
+- pretty-json@1.6.3
+- react@0.16.2
```
Comment (chriswhawkins) [inline ide/atom/README.md]: A couple of others that I have in my list that are kind of handy:
- atom-beautify (@0.29.17)
- pigments (@0.39.1)
- tabs-to-spaces (@1.0.3)

File: ide/atom/README.md (hunk 1 above)
Comment (chriswhawkins) [reply ide/atom/README.md:7]: ☝️ suspicious non-member

File: ide/atom/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+apm install <package>
+apm uninstall <package>
+apm list       # List installed packages
+apm upgrade    # Upgrade and/or update out-of-date packages
+```
+
+## Feedback
+
+- Struggles with large JSON files (60k lines)
```
Comment (chriswhawkins) [inline ide/atom/README.md]: Struggles with large files period in my experience

**Review by chriswhawkins — APPROVED** [review]
Couple of comments to chew on... Looks good. Appreciate you doing this kind of stuff!

File: ide/atom/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+#### Installation
+
+```
+apm install atom-beautify busy-signal editorconfig fonts highlight-selected intentions linter linter-eslint linter-stylelint linter-ui-default minimap pigments pretty-json react tabs-to-spaces
+```
+
+## Shortcuts
+
+- [Mac](./mac.md)
```
Comment (tborres) [inline ide/atom/README.md]: Should probably have a single page of shortcuts that displays the shortcut keys for mac and windows.

File: ide/atom/mac.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# [Atom](../atom)
+
+## Duplicate selection: <kbd>⌘</kbd><kbd>d</kbd>
```
Comment (tborres) [inline ide/atom/mac.md]: 👍 I actually never use this shortcut, but I need to start.

It's <kbd>ctrl</kbd><kbd>d</kbd> in windows

File: ide/atom/mac.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+**First behavior:** Place your cursor within what you want to select, and press the shortcut. It will select the full alphanumeric sequence as your first selection. When you press it again to duplicate the selection, it defaults to selecting only **whole-word occurrences** of your current selection:
+
+![dup_selection_whole_word](http://i.imgur.com/0LqDNdt.png)
+
+**Second behavior:** By dragging to highlight, or double clicking, it will include each occurrence of that alphanumeric sequence, regardless of its context:
+
+![dup_selection_each_occurrence](http://i.imgur.com/evLAe1B.png)
+
+## Duplicate line: <kbd>⌘</kbd><kbd>Shift</kbd><kbd>d</kbd>
```
Comment (tborres) [inline ide/atom/mac.md]: It's <kbd>ctrl</kbd><kbd>shift</kbd><kbd>d</kbd> in windows

File: ide/atom/mac.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Place your cursor anywhere, and it duplicates the current line. Your cursor remains in the lower line.
+
+## Open Project File: <kbd>⌘</kbd><kbd>Shift</kbd><kbd>t</kbd>, also <kbd>⌘</kbd><kbd>Shift</kbd><kbd>p</kbd>
```
Comment (tborres) [inline ide/atom/mac.md]: Windows: <kbd>ctrl</kbd><kbd>t</kbd> or <kbd>ctrl</kbd><kbd>p</kbd>

File: ide/atom/mac.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Press it, and you can start typing the name of the file immediately.
+
+## Move selected line(s) up and down: <kbd>⌘</kbd><kbd>Ctrl</kbd><kbd>↑</kbd>/<kbd>↓</kbd>
```
Comment (tborres) [inline ide/atom/mac.md]: Windows: <kbd>ctrl</kbd><kbd>↑</kbd><kbd>ctrl</kbd><kbd>↓</kbd>

File: ide/atom/mac.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+## Needed
+- Split panes
```
Comment (tborres) [inline ide/atom/mac.md]: Windows: <kbd>ctrl</kbd><kbd>k</kbd> and then arrow in the direction to split (e.g. <kbd>→</kbd>)

File: ide/atom/mac.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+- Toggle tree view(?) (left side project explorer)
```
Comment (tborres) [inline ide/atom/mac.md]: Windows: <kbd>ctrl</kbd><kbd>\\</kbd>

**Review by tborres — CHANGES_REQUESTED** [review]
Looks great. Some suggested changes to the shortcuts and then it's ready to go.

**Pull request merged**